### PR TITLE
Add ignore_subs setting to manual PP

### PR DESCRIPTION
**Pull request by fernandog**
If one episode is hanging in PP folder and never gets PP because there is no subs available, this setting will override the associated subtitle check and it will PP the file

Also add some missing descriptions to the settings:

image

### CODE REVIEW

**Comment by fernandog** [comment]
@ratoaq2 when you have time can you review this please?

File: sickbeard/processTV.py [hunk 1]
```diff
@@ -543,7 +544,7 @@ def already_postprocessed(dirName, videofile, force, result):  # pylint: disable
     return False
 
 
-def process_media(processPath, videoFiles, nzbName, process_method, force, is_priority, result):  # pylint: disable=too-many-arguments
+def process_media(processPath, videoFiles, nzbName, process_method, force, is_priority, result, ignore_subs):  # pylint: disable=too-many-arguments
```
Comment (ratoaq2) [inline sickbeard/processTV.py]: The result should be the last parameter. It makes more sense, IMO

Comment (fernandog) [inline sickbeard/processTV.py]: done

**Comment by ratoaq2** [comment]
Approved

**Comment by labrys** [comment]
@fernandog Make sure you add PRs to the milestone before merging them.

**Pull request merged**